docs/tcp_LAN_WAN_transport.rst

```diff
@@ -0,0 +1,43 @@
+TCP Transport
+==============
+
+Fast RTPS transport layer could be adapted to specific network requirements. UDP transport is optimal for localhost applications but in order to interact within a LAN or WAN a TCP transport may be required. Here we explain how Shapes Demo should be set up in order to fit some network specific layouts.
+
+LAN configuration
+-----------------
+
+This is a common scenario where several computers share the same LAN network. In this case one of the computers should set up Shapes Demo as a server and all the others as clients. Let's suppose that the server computer LAN IP address is 192.168.2.49 then all clients instances of Shapes Demo in the other computers must specify this IP Server address. In this case we select as TCP port 5100 but any other available TCP port is valid. The server computer firewall must allow inbound traffic on the selected port.
```

Comment:
The common scenario is not this. Usually in LAN is used UDP, but we offer the example of TCP in a LAN network.